lookout/style/typos/ranking.py

```diff
@@ -24,41 +25,33 @@ class CandidatesRanker(Model):
     VENDOR = "source{d}"
     DESCRIPTION = "Model that ranks candidates according to their probability to fix the typo."
     LICENSE = DEFAULT_LICENSE
-    DEFAULT_TRAIN_ROUNDS = 4000
-    DEFAULT_EARLY_STOPPING = 200
-    DEFAULT_BOOST_PARAM = {"max_depth": 6,
-                           "eta": 0.03,
-                           "min_child_weight": 2,
-                           "silent": 1,
-                           "objective": "binary:logistic",
-                           "nthread": 16,
-                           "subsample": 0.5,
-                           "colsample_bytree": 0.5,
-                           "alpha": 1,
-                           "eval_metric": ["error"]}
-
-    def __init__(self, **kwargs):
-        """Initialize a new instance of CandidatesRanker class."""
+
+    def __init__(self, config: Optional[Mapping[str, Any]] = None, **kwargs):
+        """
+        Initialize a new instance of CandidatesRanker class.
+
+        :param config: Ranking configuration, options:
```

Comment:
Missing "\" in the ends of lines.

Reply:
Missing what exactly? "\" will put everything in one line afaik

Reply:
Oh i see:)